common/lib/xmodule/xmodule/modulestore/mixed.py

```diff
@@ -322,6 +322,21 @@ def get_course(self, course_key, depth=0, **kwargs):
             return None
 
     @strip_key
+    def do_index(self, course_key, depth=0, **kwargs):
+        """
+        restarts index on a course with the course_id. If no such course exists,
+        it returns None
+
+        :param course_key: must be a CourseKey
+        """
+        assert(isinstance(course_key, CourseKey))
```

Comment:
`assert` is a keyword, not a function, and is generally used without parentheses.
```python
assert isinstance(course_key, CourseKey)
```